Engine/source/scene/pathManager.cpp

```diff
@@ -299,15 +303,33 @@ void PathManager::getPathPosition(const U32 id,
 
    // Ok, query holds our path information...
    F64 ms = msPosition;
-   if (ms > mPaths[id]->totalTime)
-      ms = mPaths[id]->totalTime;
+
+   //Looping vs. non-looping splines
+   F32 msTotal;
```

Comment:
Shouldn't this be a `U32` instead?

```suggestion
   U32 msTotal;
```